host_core/lib/host_core/control_interface/server.ex

```diff
@@ -68,129 +75,138 @@ defmodule HostCore.ControlInterface.Server do
     end
   end
 
+  ### LINKDEFS
+  # These requests are targeted at one host per lattice, changes made as a result
+  # are emitted to the appropriate stream and cached
+
+  # Put a link definition
+  # This will first store the link definition in memory, then publish it to the stream
+  # then publish it directly to the relevant provider via the RPC channel
+  defp handle_request({"linkdefs", "put"}, body, _reply_to) do
+    ld = Jason.decode!(body)
+
+    HostCore.Linkdefs.Manager.put_link_definition(
+      ld["actor_id"],
+      ld["contract_id"],
+      ld["link_name"],
+      ld["provider_id"],
+      ld["values"]
+    )
+
+    {:reply, ""}
+  end
+
+  # Remove a link definition
+  # This will first remove the link definition from memory, then publish the removal
+  # message to the stream, then publish the removal directly to the relevant provider via the
+  # RPC channel
+  defp handle_request({"linkdefs", "del"}, body, _reply_to) do
+    ld = Jason.decode!(body)
+
+    HostCore.Linkdefs.Manager.del_link_definition(
+      ld["actor_id"],
+      ld["contract_id"],
+      ld["link_name"]
+    )
+
+    {:reply, ""}
```

Comment:
For our other operations we've been following a pattern of responding with a reply object, usually just indicating if the operation was successful and if not then a failure message. Can we indicate the status of the del_link / put_link in that reply?

For strongly typed languages like Rust, we may want to consider standardizing the `ack` that we do respond with, e.g. just an object with success and error message